### ISSUE TITLE
Toolbar cursor and placeholder not shown next to search icon (Intermittent)

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#22319]
!Screenshot 2024-10-01 at 1.07.50 PM.png|width=1062,height=2002,alt="Screenshot 2024-10-01 at 1.07.50 PM.png"!

!Screenshot 2024-10-01 at 1.07.44 PM.png|width=1062,height=2002,alt="Screenshot 2024-10-01 at 1.07.44 PM.png"!

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-10199)

**Comment by data-sync-user** [comment]
➤ ih-codes commented:

Here is a video I started recording when this bug occurred. I tried to see if creating a new tab would “fix” it, but it didn’t “fix” until I selected an existing non-home tab in this case.

!bug toolbar spacing until tab select.mp4|width=1178,height=2556,alt="bug toolbar spacing until tab select.mp4"!

The bug occurred as I was opening multiple new tabs and tapping the top site tiles and testing searches. Probably not related, but I was using a non-default search engine as well (DDG).
